cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -613,8 +613,20 @@ public void extractTar(Path file, Path targetDir, TarCompression compression) {
   public void extractJar(Path file, Path targetDir) {
 
     this.context.trace("Unpacking JAR {} to {}", file, targetDir);
+    // calculate the size of archive for the progressbar max size
+    long sizeOfArchive = 0;
```

Comment:
This workaround counting the size before init the progressbar works for ZIPs but JARs, as for example gcviewer.jar, have the size of -123

Probably that's still a bug in `extractJar()`

Reply:
progressbar for JAR extraction removed for now